packages/core/src/components/menu/_menu.scss

```diff
@@ -342,6 +346,15 @@ Styleguide components.menu.css.header
       }
     }
 
+    &:not(.pt-disabled) {
```

Comment:
fair, we'll need to reset all the icons and text too, but i guess that's only 1 more level down anyway - updated